docs/api/en/core/Layers.html

```diff
@@ -72,6 +72,13 @@ <h3>[method:Boolean test]( [param:Layers layers] )</h3>
 			Returns true if this and the passed *layers* object have at least one layer in common.
 		</p>
 
+		<h3>[method:Boolean testInteger]( [param:Integer layer] )</h3>
+		<p>
+			layer - an integer from 0 to 31.<br /><br />
```

Comment:
0 to 31?

Reply:
It seems from the doc that 

> A Layers object assigns an Object3D to 1 or more of 32 layers numbered 0 to 31

Should I change something ?

Reply:
I think @felixmariotto was correct here. Valid channel are in the integer range `[0,31]`.

BTW: I would rename the method to `testChannel()` and add the discription: `Whether the given channel is enabled or not.